src/web/client/common/portalSchema.ts

```diff
@@ -0,0 +1,6264 @@
+export const portal_schema_data = {
+    "entities":{
+       "organization":[
```

Comment:
let's rename this to rootEntity or commonEntity or fallbackEntity
organization is very specific to Dataverse. In case of GitHub it won't be relevant.

Reply:
On second thought I feel dataSourceProperties would be even more relevant and aligned with our terminologies.

Reply:
done